apps/epic-web/next.config.js

```diff
@@ -60,6 +60,21 @@ const nextConfig = {
         destination: '/contributors/:slug*',
         permanent: true,
       },
+      {
+        source: '/conf',
+        destination: '/conf/2025',
+        permanent: true,
```

Comment:
```suggestion
        permanent: false,
```

I don't think it is since we will change it again later probably

Reply:
good point